src/app/components/icons/ProgressIconCard.tsx

```diff
@@ -34,6 +36,14 @@ export const ProgressIconCard: React.FC = (): JSX.Element => {
                     </Typography>
                 </Toolbar>
             </AppBar>
+            <Toolbar style={{ justifyContent: 'flex-end' }}>
+                <FormControlLabel
+                    control={<Checkbox color="primary" onClick={(): void => setIsOutlined(!isOutlined)} />}
+                    checked={isOutlined}
+                    label={'View Outlined Icon'}
```

Comment:
```suggestion
                    label={'View Outlined'}
```

or View Outlined _Icons_